webcompat/static/js/lib/editor.js

```diff
@@ -32,23 +32,23 @@ issues.CategoryView = Backbone.View.extend({
 */
 issues.CategoryEditorView = Backbone.View.extend({
   isOpen: false,
-  className: "wc-CategoryEditor js-CategoryEditor",
+  className: "label-editor js-CategoryEditor",
   events: {
     "change input[type=checkbox]": "updateView",
     "click button": "closeEditor",
     keyup: "closeEditor",
-    "keyup .wc-CategoryEditor-search": "filterItems",
-    "keyup .wc-CategoryEditor-list-item": "checkUncheckItems",
-    "keydown .wc-CategoryEditor-search": "focusSaveClose",
-    "keydown .wc-CategoryEditor-list-item": "removeFocus",
-    "keydown .wc-CategoryEditor-list-item:visible:last": "backToTop"
+    "keyup .label-editor-header .form-field": "filterItems",
```

Comment:
This change will break tests in labels-auth.js and milestones-auth.js. Can we update the selectors there?

(i mean, i guess nearly all tests are broken anyways, but might as well do it now. :P)

Reply:
(bonus points for using a `.js-foo` class in the template so we never have to change it again...)

Reply:
Ah, I see you already did that. Cool. So we want to update the tests to use `.js-MilestoneEditor-search`. 👍

Reply:
Removing the last `wc` classes are on my list and moving them to a `js-` class naming convention as it seems this is what we (as a project) agreed on.